integration/activemq-vertx-integration/src/main/java/org/apache/activemq/artemis/integration/vertx/ActiveMQVertxLogger.java

```diff
@@ -38,7 +38,7 @@
  * so an INFO message would be 191000 to 191999
  */
 @MessageLogger(projectCode = "AMQ")
-public interface ActiveMQVertxLogger extends BasicLogger {
+interface ActiveMQVertxLogger extends BasicLogger {
```

Comment:
Because I don't want that anyone else ever is able to use this logger in other packages/tests. Currently it is used within the Vert.x package only and in my opinion it should stay that modular.